config/test.exs

```diff
@@ -54,3 +54,6 @@ config :code_corps,
 
 config :code_corps, :cloudex, CloudexTest
 config :cloudex, api_key: "test_key", secret: "test_secret", cloud_name: "test_cloud_name"
+
+# Plug.Conn errors on uppercased event names, so we need to mock it here
+config :code_corps, github_event_header_name: "x-github-event"
```

Comment:
Couldn't resist googling and that's just...

Ok, it's supposed to be a standard and a convention, but forcing it like this...